src/utility/index.jsx

```diff
@@ -344,12 +351,64 @@ export const transformations = {
         `x${row.party_size}`,
       ];
     };
+    const won = field === isRadiant(row.player_slot);
+    const getColor = (result) => {
+      if (result === null || result === undefined) {
+        return constants.colorMuted;
+      }
+      return won ? constants.colorGreen : constants.colorRed;
+    };
+    const getString = (result) => {
+      if (result === null || result === undefined) {
+        return strings.td_no_result;
+      }
+      return won ? strings.td_win : strings.td_loss;
+    };
+    const skillDotsStyle = {
+      height: '5px',
+      width: '5px',
+      backgroundColor: constants.lightGray,
+      display: 'inline-block',
+      marginLeft: '3px',
+      marginBottom: '2px',
+      transform: 'rotate(45deg)',
+    };
+    const getSkill = (skill) => {
+      const skillDots = [];
+      if (skill) {
+        for (let i = 0; i < 3; i += 1) {
+          skillDots.push(<div style={{ ...skillDotsStyle, opacity: skill - i > 0 ? '1' : '0.3' }} />);
+        }
+      }
+      return skillDots;
+    };
+    let normalOrRanked;
+    if (row.lobby_type === 7 || row.lobby_type === 0) {
+      const style = {
+        fill: row.lobby_type === 7 ? constants.yelor : constants.lightGray,
+        height: '13px',
+        position: 'relative',
+        left: '2px',
+        top: '1px',
+      };
+      normalOrRanked = <IconRanking style={style} data-tip={strings[`lobby_type_${row.lobby_type}`]} data-delay-show="300" data-offset="{'right': 3, 'top': 3}" />;
```

Comment:
Don't really like the ranked icon (not very clear to me), maybe we can keep text for that?